README.md

```diff
@@ -1 +1,20 @@
-# microverse_trial_3
\ No newline at end of file
+**Microverse_trial_3**
+
+Portfolio website mobile version
+Home section
+
+**Built With**
+
+- HTML
+- CSS
+- VS Code
+  **Authors**
+  :silhouette: **Shahnawaz Ali**
+- GitHub: [@githubhandle](https://github.com/shahnawaza75)
+- Twitter: [@twitterhandle](https://twitter.com/RjShahnawaza75)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/shahnawaz-ali-a24b72204)
```

Comment:
It will be nice if you add your Username to your link text for your socials. `@shahnawaza75`

Reply:
- [ ] Nice job being polite, giving reasons for the requested change, and phrasing your comments as suggestions